app/_src/gateway-operator/production/upgrade/data-plane/rolling.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Rolling Upgrades
+---
+
+## Using DataPlane
+
+{:.note}
+> This method is only available when running in [hybrid mode](/gateway-operator/{{ page.release }}/topologies/hybrid/)
+
+To change the image used for your `DataPlane` resources, set the `spec.deployment.podTemplateSpec.spec.containers[].image` field in your resource:
+
+```bash
+kubectl edit dataplane dataplane-example
+```
+
+Once the resource is saved, Kubernetes will perform a rolling upgrade of your pods.
```

Comment:
Here there is written "pods", but check the last line of the file above there is "`Pod`s". Do we have any consistency guideline for the whole documentation that advise one particular way?

Reply:
```suggestion
Once the resource is saved, Kubernetes will perform a rolling upgrade of your `Pod`s.
```

Reply:
We don't, but we should have. @Rajakavitha1 please make a recommendation in a followup PR